<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ai_tab</class>
 <widget class="QWidget" name="ai_tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea_5">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>841</width>
        <height>464</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>AI:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="ai_select_combo"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ai_name_edit"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="margin">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Condition:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="ai_condition_combo">
                   <item>
                    <property name="text">
                     <string>No Condition</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_13">
               <property name="margin">
                <number>5</number>
               </property>
               <item row="1" column="1">
                <widget class="QComboBox" name="ai_action_jump_combo">
                 <item>
                  <property name="text">
                   <string>left</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>center</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>right</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>to top</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>to ground</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>to player</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_52">
                 <property name="text">
                  <string>Jump:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_56">
                 <property name="text">
                  <string>Attack:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="ai_action_attack_combo"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_58">
                 <property name="text">
                  <string>Move:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="ai_action_move_combo">
                 <item>
                  <property name="text">
                   <string>to left</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>to right</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>cross screen</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>follow player</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>return to last position</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>player direction</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_59">
                 <property name="text">
                  <string>Wait:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <spacer name="verticalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="4" column="1">
                <widget class="QSpinBox" name="ai_action_wait_spinbox"/>
               </item>
               <item row="1" column="2">
                <widget class="QRadioButton" name="action_radio_jump">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="shortcut">
                  <string>Ctrl+S</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QRadioButton" name="action_radio_attack">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QRadioButton" name="action_radio_move">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="2">
                <widget class="QRadioButton" name="action_radio_wait">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="margin">
              <number>5</number>
             </property>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_10">
               <property name="text">
                <string>Add -&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_pushButton">
               <property name="text">
                <string>&lt;- Remove</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QListView" name="ai_actions_list"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
